pkg/auth/model/model.go

```diff
@@ -308,15 +292,15 @@ func protoFromStatements(s *Statements) []*StatementData {
 }
 
 func KVUserToGroup(groupDisplayName string, userName string) string {
-	return kv.FormatPath(authPrefix, "groups", groupDisplayName, "users", userName)
+	return kv.FormatPath("groups", groupDisplayName, "users", userName)
 }
 
 func KVPolicyToUser(userName string, policyDisplayName string) string {
-	return kv.FormatPath(authPrefix, "users", userName, "policies", policyDisplayName)
+	return kv.FormatPath("users", userName, "policies", policyDisplayName)
 }
 
 func KVPolicyToGroup(groupDisplayName string, policyDisplayName string) string {
-	return kv.FormatPath(authPrefix, "groups", groupDisplayName, "policies", policyDisplayName)
+	return kv.FormatPath("groups", groupDisplayName, "policies", policyDisplayName)
```

Comment:
1. Use consts for path elements
2. Paths to different objects / secondary prefixes should not have a common subpath
3. Please put all Path functions together and use the same naming convention e.g.: "PolicyGroupPath"
4. Remove KV prefix from function names - it's redundant